docs/UserGuide.md

```diff
@@ -103,15 +103,16 @@ Expected output:
 
 Edits an existing contact with a given name. This command supports changing the name, phone number and email.
 
-Format: `edit NAME [-n NEW_NAME] [-p NEW_PHONE_NUMBER] [-e NEW_EMAIL]`
+Format: `edit COURSEMATE [-n NEW_NAME] [-p NEW_PHONE_NUMBER] [-e NEW_EMAIL]`
 
 Parameters:
 - `NAME`: Name of the existing contact. Accepts aliases (substrings or in hashtag (#) notation).
 - `NEW_NAME` (optional): New name of the contact. It can be any string except that it cannot start with the hashtag (#) symbol, and words cannot start with the hyphen (-) character.
 - `NEW_PHONE_NUMBER` (optional): New phone number of the contact. Accepts strings with numeric characters only.
 - `NEW_EMAIL` (optional): New email of the contact. Accepts any strings without spaces.
 
-ℹ
+<box type="tip" seamless>
+
 **Tip:**
 
 When supplying a name to identify an existing contact, you can use the following shortcuts:
```

Comment:
Consider using `Instead of supplying a full name...` other than `When supplying a name...` since it's no longer called `NAME`.